R/EpivizTreeData-class.R

```diff
@@ -552,17 +556,27 @@ EpivizTreeData$methods(
     
     
     measurements <-  metadata(.self$.object)$tsne
+    #print("123")
+    #print(measurements)
+    
     data <- list()
+    level<- .self$.levelSelected
+    i<- 1
     for (col in rownames(metadata(.self$.object)$tsne)) {
       temp    <-
         list(
           sample_id = col,
           PC1 = unname(measurements[col, 1]),
-          PC2 = unname(measurements[col, 2])
+          PC2 = unname(measurements[col, 2]),
+          cluster = unname(treeviz@colData@listData[[level]][i])
```

Comment:
1. `treeviz` doesn't exist, replace it with `.self$.object`
2. not sure what you are accessing here, is it `hierarchy_tree` or `nodes_table` ? 
    if its hierarchy_tree, use getters/setters here either `.self$.graph@hiearchy_tree` or `colData(.self$.object)@hierarchy_tree`

Reply:
also didn't we change "cluster" to "name" ?

Reply:
Lets rename `PC1` to `dim1` and `PC2` to `dim2`

Reply:
There is a `listData` field under `colData`, I am using that here. I didn't use `hierarchy_tree` since you suggested it will be slow